packages/salesforcedx-utils-vscode/src/context/workspaceContextUtil.ts

```diff
@@ -103,6 +104,14 @@ export class WorkspaceContextUtil {
         defaultUsernameOrAlias !== this._username
           ? defaultUsernameOrAlias
           : undefined;
+      try {
+        const connection = await this.getConnection();
+        this._orgId = connection?.getAuthInfoFields().orgId;
+      } catch (error) {
+        console.log(
+          `There was an problem getting the orgId of the default org: ${error}`
```

Comment:
this.getConnection() throws so...

Reply:
nit: should we set _orgId to '' or undefined in this case of a catch?  Looking to avoid having an orgId for the previously auth'd org if this throws when getting the orgId for whatever reason.

Reply:
Yep, good call

Reply:
I put this in and added a test for it